internal/repository/halls/hall.go

```diff
@@ -0,0 +1,55 @@
+package halldb
+
+import (
+	"database/sql"
+)
+
+type Repository struct {
+	DB *sql.DB
+}
+
+type Resource struct {
+	ID    int  `json:"ID"`
+	VIP   bool `json:"VIP"`
+	Seats int  `json:"seats"`
+}
+
+//Create new Hall in DB
+func (r *Repository) Create(hall Resource) error {
+	insertHall := `insert into halls("vip","seats") values($1,$2)`
+	_, err := r.DB.Exec(insertHall, hall.VIP, hall.Seats)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+//Retrieve Hall from DB
+func (r *Repository) Retrieve(id int64) (dbHall Resource, e error) {
+
+	rows, err := r.DB.Query(`SELECT * FROM halls WHERE "id" = $1`, id)
```

Comment:
Try squirell

Reply:
i need to test DB forwarding with working connection, then will try use sqirell, cause last time it didnt work. And i guess debug errors in db connection and server logic at the same time isnt very easy, wanna try solve one problem, and then try other

Reply:
Ok, leave it for now. I've created a [ticket](https://github.com/DarkJediDJ/cinetickets/projects/1#card-76777466), which is first in your queue